test/spec/unwind.txt

```diff
@@ -1,13 +0,0 @@
-;;; TOOL: run-interp-spec
```

Comment:
Ditto my other comment, I think this file should stay.

Reply:
Done